modules/ai/assets/js/editor/layout-content.js

```diff
@@ -6,9 +7,13 @@ import UpgradeChip from './components/upgrade-chip';
 import useUserInfo from './hooks/use-user-info';
 import WizardDialog from './components/wizard-dialog';
 import LayoutDialog from './pages/form-layout/components/layout-dialog';
+import { Alert } from '@elementor/ui';
+import useIntroduction from './hooks/use-introduction';
 
 const LayoutContent = ( { onClose, onConnect, onGenerationStart, onGenerationEnd, onInsert, onSelect } ) => {
 	const { isLoading, isConnected, isGetStarted, connectUrl, fetchData, hasSubscription, credits, usagePercentage } = useUserInfo();
+	const { isViewed, markAsViewed } = useIntroduction( 'e-ai-builder-no-images-info' );
+	const [ showInfoAlert, setShowInfoAlert ] = useState( () => ! isViewed );
```

Comment:
pretty sure you can use only the introduction state

Reply:
Done.